tests/retrieve/test_access.py

```diff
@@ -307,14 +307,23 @@ def test_get_obs_surface_cloud(mocker, monkeypatch):
 
 
 def test_get_obs_column():
```

Comment:
Could we split this out into two tests? - keep `test_get_obs_column` with the necessary tweaks (e.g. `xch4` is not `mf`) but then have a second test called something like `test_get_obs_column_max_level` (wordy!) which just tests the use of `max_level`. The benefit of doing that is if e.g. just the max_level feature starts failing it would be easier to pinpoint the issue.